### DISCUSSION TITLE
How to customise colours?

### DISCUSSION THREAD

**Discussion by pro-sumer** [Eugeny/tabby#6177] [Q&A] [answered]
In my current terminal (iTerm) I use a custom color scheme based on [Monokai Pro](https://monokai.pro) and I want to use the same colours in Tabby. However, immediately after changing a (single) color in the Color scheme section of the Settings and saving that change the colour reverts to the default defined by Tabby.

Since I'm new to Tabby I'm not sure whether this is a user error or a bug...

How can I customise colours?

**Comment by Eugeny** [comment, marked as answer]
This is a bug that's already fixed in the nightly build (https://nightly.link/Eugeny/tabby/workflows/build/master)

**Comment by lawweiliang** [comment]
@pro-sumer 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
